index.bs

```diff
@@ -1368,6 +1368,17 @@ must be taken to strike the right balance between restricting what the API can a
 having the API be useful. After all, this API intentionally lets the user use websites to interact
 with some of their most private personal data.
 
+Examples of directories that user agents might want to restrict as being
+<dfn>too sensitive or dangerous</dfn> include:
+
+* The directory or directories containing the user agent itself.
+* Directories where the user agent stores [=site storage=].
+* Directories containing system files (such as `C:\Windows` on Windows).
+* Directories such as `/dev/`, `/sys`, and `/proc` on Linux that would give access to low-level devices.
+* A users entire "home" directory.
+  Individual files and directories inside the home directory should still be allowed,
+  but user agents should not generally let users give blanket access to the entire directory.
+
```

Comment:
WDYT about mentioning the default directory for downloads? (`<a download>` and downloads with `Content-Disposition: attachment`)